format/index.md

```diff
@@ -189,10 +189,12 @@ be represented under the "links object".
 
 #### Complex Attributes <a href="#document-structure-resource-object-complex-attributes" id="document-structure-resource-object-complex-attributes"></a>
 
-"[Complex attributes]" are [attributes] whose value is an object or array with
-any level of nesting. An object that constitutes or is contained in a complex
-attribute **MUST** reserve the `id`, `type`, `links`, and `meta` members for future
-use.
+"[Complex attributes]" are [attributes] whose value is an object or array
+([example](/examples/#complex-attributes)).
+
+JSON API permits embedding arbitrary data structures as complex attributes in
+resource objects. However, any object in a complex attribute **MUST** reserve
```

Comment:
@ethanresnick Maybe. The phrase was my brainchild, but then I started doubting its necessity. If you think it adds clarity, I'll put it back. It just doesn't seem like "any object in a complex attribute" could realistically be misinterpreted.

While we're at it, the first paragraph used to have another clause:

> "Complex attributes" are attributes whose value is an object or array, which in turn may contain values of any type.

But then I thought it might be overkill given the second paragraph. What do you think?